src/mscorlib/ref/mscorlib.cs

```diff
@@ -2483,7 +2506,8 @@ public sealed partial class LoaderOptimizationAttribute : System.Attribute
     [System.Runtime.InteropServices.ComVisibleAttribute(true)]
     public sealed partial class LocalDataStoreSlot
     {
-        internal LocalDataStoreSlot() { }
+        internal LocalDataStoreSlot(object mgr, int slot, long cookie) { }
```

Comment:
The internal methods have no impact in this file - I do not think you will be able to call them from outside of corelib. None of this LocalDataStoreSlot stuff should be coming from corelib. It should be all in corefx.